### PR TITLE
2024 discourse announcement

### PR DESCRIPTION
**Pull request by GuillaumeDesforges**

### CODE REVIEW

**Review by fricklerhandwerk — APPROVED** [review]

File: 2024/COMMUNICATION.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+## Discourse announcement
+
+Dear Nix Community,
+
+It is with excitement that we launch 2024's Nix Survey!
+
+https://survey.nixos.org/2024
+
+We are conducting this survey to improve our understanding of the community and what everyone cares about.
+We hope to use your responses to support Nix Teams in best serving the community.
```
Comment (refroni) [inline 2024/COMMUNICATION.md]: (Slight wording) - We plan to use your responses to support Nix Teams to better serve the community.

File: 2024/COMMUNICATION.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+As always, the survey is **anonymous**, and aggregated statistics and major findings will be published on Discourse.
```
Comment (refroni) [inline 2024/COMMUNICATION.md]: As always, the survey is **anonymous**. Aggregated statistics and major findings will be published on Discourse.

File: 2024/COMMUNICATION.md (hunk 1 above)
Comment (GuillaumeDesforges) [reply 2024/COMMUNICATION.md]:
```suggestion
We plan to use your responses to support Nix Teams to better serve the community.
```

File: 2024/COMMUNICATION.md (hunk 2 above)
Comment (GuillaumeDesforges) [reply 2024/COMMUNICATION.md]:
```suggestion
As always, the survey is **anonymous**
Aggregated statistics and major findings will be published on Discourse.
```

**Pull request merged**